Portfolio/index.css

```diff
@@ -1,15 +1,20 @@
-/* Write your code below for PORTFOLIO */
 * {
     margin: 0px;
     padding: 0px;
     font-family: "Revalia", sans-serif;
     color: #00D8FF;
+    box-sizing: border-box;
 
 }
 

```

Comment:
The .about class is not closed properly. Add a closing curly brace after line 16.